example_dags/example_dag_fivetran.py

```diff
@@ -55,38 +25,34 @@
     transfer_fivetran_with_connector_id = UniversalTransferOperator(
         task_id="transfer_fivetran_with_connector_id",
         source_dataset=File(path=f"{s3_bucket}/uto/", conn_id="aws_default"),
-        destination_dataset=Table(name="fivetran_test", conn_id="snowflake_default"),
+        destination_dataset=Table(name="fivetran_ankit_test", conn_id="snowflake_fivetran_conn"),
```

Comment:
Can we keep a generic name?

Reply:
[Done](https://github.com/astronomer/apache-airflow-provider-transfers/pull/62/commits/9fd6e7f7850c27f22e10d44a17e5f5c0744a15f9)